Program.cs

```diff
@@ -9,15 +9,47 @@ class Program
         int saldoLimite = 20000;
         bool MaximoCobro(TarjetaCredito credit) //Si el credito es menor al limite de saldo devuelve verdadero si no, devuelve falso
         {
-            if(credit.Saldo < saldoLimite)
+            if(credit.Saldo < Limite)
             {
                 return true;
             }
 
             return false;
         }
-        
-        
+        bool validarTarjeta(TarjetaDebito debit)
+        {
+            if(debit.NumeroDeTarjeta.leght != 16)
```

Comment:
error CS1061: 'long' does not contain a definition for 'leght' and no accessible extension method 'leght' accepting a first argument of type 'long' could be found (are you missing a using directive or an assembly reference?)